src/babylon.scene.ts

```diff
@@ -339,6 +339,9 @@
         */
         public meshes = new Array<AbstractMesh>();
 
+        // Contain all canvas2D used in the scene
+        public canvas2ds = new Array<Canvas2D>();
+

```

Comment:
Please, no Canvas2D stuff in Scene.ts.
Instead you should rely on a public get Canvas(): Array<Canvas2D> in Canvas2D class with a private static class behind.